composites/Plugin/CornerstoneContent/components/CornerstoneToggle.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import Toggle from "../../Shared/components/Toggle";
+import { __ } from "@wordpress/i18n";
+
+const Cornerstone = styled.div`
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin-top: 5px;
+
+	label { 
+		margin-right: 10px;
+		flex-shrink: 0;
+		max-width: 75%;
+	}
+`;
+
+class CornerstoneToggle extends React.Component {
+	/**
+	 * Renders the CornerstoneToggle component.
+	 *
+	 * @returns {ReactElement} the CornerstoneToggle component.
+	 */
+	render() {
+		return (
+			<Cornerstone>
+				<Toggle ariaLabel="Mark this post as cornerstone content" id="Cornerstone Toggle"
```

Comment:
the id should be lowercase and without spaces (it's used for HTML attributes). Also, it can't be hardcoded. IDs must be unique in a page, otherwise it's impossible to use more than one toggle.

Reply:
True, in the toggle it is a prop we can set, but in the cornerstone toggle we keep it hard coded for now because there is only one cornerstone toggle.